### DISCUSSION TITLE
Is the `edr-cache-api` extension included in the Tractus-EDC?

### DISCUSSION THREAD

**Discussion by efiege** [eclipse-tractusx/tractusx-edc#1381] [Q&A] [answered]
The core-EDC (v0.7.1) provides the [edr-cache-api](https://github.com/eclipse-edc/Connector/tree/v0.7.1/extensions/control-plane/api/management-api/edr-cache-api) extension to manage EDRs using the  management api.

Are those endpoints also available in the Tractus-EDC?

**Comment by wolf4ood** [comment, marked as answer]
Hi @efiege 

we used a variant of EDR api which supports token renewals. The API docs is here

https://app.swaggerhub.com/apis/eclipse-tractusx-bot/tractusx-edc/0.7.3#/Control%20Plane%20EDR%20Api/

**Reply by efiege** [reply]
Hey @wolf4ood,

thanks for the fast answer!

I see there are /v1 and /v3 versions in the core-EDC and the /v2 version in the Tractus-EDC. Is it planned to transition to the /v3 version in the future?

**Reply by wolf4ood** [reply]
The `Tractus-EDC` will follow it's own versioning scheme since it's a custom api for managing EDRs with refresh support

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
